### PR TITLE
Connect Webview Create -- Redirect to Link

### LINKED ISSUE #49: Connect Webview Create -- Redirect to Link
It would be super nice to prompt the user to redirect them in their browser to the connect webview url that gets generated after invoking a create command. Or at least offer to copy the url in the clipboard.

```sh
➜  ~ seam connect-webviews create

✔ [/connect_webviews/create] Parameters › [Make API Call] /connect_webviews/create


/connect_webviews/create
Request Params:
{}


[200]
{
  connect_webview: {
    url: 'https://connect.getseam.com/connect_webviews/view?connect_webview_id=3a966edd-9b46-41c1-a330-b899891b943e&auth_token=CjruEU1FABFwf1ZkpGEeVaUXvN94ugjpE',
    status: 'pending',
```

### PR DESCRIPTION
**Pull request by mastafit**

### CODE REVIEW

File: cli.ts [hunk 1]
```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
 
   logResponse(response)
 
+  if (isEqual(selectedCommand, ["connect-webviews", "create"])) {
```
Comment (seveibar) [inline cli.ts:173]: use the response instead of the command

```suggestion
  if (response.data.connect_webview) {
```

File: cli.ts [hunk 2]
```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
...
+    if (response.data && response.data.connect_webview && response.data.connect_webview.url) {
+      const url = response.data.connect_webview.url;
+      
+      if (process.env.INSIDE_WEB_BROWSER !== '1') {
+        const { action } = await prompts({
+          type: 'select',
```
Comment (seveibar) [inline cli.ts:179]:
```suggestion
          type: 'confirm',
```

File: cli.ts [hunk 3]
```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
...
+          name: 'action',
+          message: 'What do you want to do with the connect webview URL?',
```
Comment (seveibar) [inline cli.ts:181]:
```suggestion
          message: 'Would you like to open the webview in your browser?',
```

File: cli.ts [hunk 4]
```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
...
+          choices: [
+            { title: 'Open in browser', value: 'open' },
+            { title: 'Copy to clipboard', value: 'copy' },
+          ],
```
Comment (seveibar) [inline cli.ts:185]: also no need to support `copy`

```suggestion
```

File: cli.ts [hunk 5]
```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
...
+        });
+  
+        if (action === 'open') {
+          await open(url);
+        } else if (action === 'copy') {
+          clipboardy.writeSync(url);
+          console.log('URL copied to clipboard!');
+        }
+      } else {
+        console.log("Running in a web browser. Prompting is skipped.");
```
Comment (seveibar) [inline cli.ts:195]: Don't log this, it will show up for everyone creating connect webviews who says no

```suggestion
```

File: package.json [hunk 6]
```diff
@@ -54,6 +54,8 @@
   },
   "dependencies": {
     "@seamapi/http": "^0.12.0",
+    "clipboardy": "^4.0.0",
```
Comment (seveibar) [inline package.json:57]: No need to support copy to clipboard

```suggestion
```

**Review by seveibar — CHANGES_REQUESTED** [review]
see comments

**Comment by mastafit** [comment]
> see comments

Will be done

**Review by seveibar — APPROVED** [review]

**Pull request merged**